Makefile

```diff
@@ -1,70 +1,29 @@
-PINT_VERSION = 0.42.2
-PROMTOOL_VERSION = 2.42.0
-YQ_VERSION = 4.31.2
-EXTRACTED_RULE_FILE = test/promql/extracted-rules.yaml
-ALERTING_RULE_FILES = rhobs/alerting/*.yaml
+SHELL=/usr/bin/env bash -o pipefail
 
-.PHONY: all
-all: prepare sync_pipenv lint test_rules pint_lint lint_yamls kustomize_build
-
-.PHONY: prepare
-prepare: pint promtool yq kustomize
-	echo "Extract Prometheus rules"
-	(./yq eval-all '. as $$item ireduce ({}; . *+ $$item)' ${ALERTING_RULE_FILES} | ./yq ".spec" ) > ${EXTRACTED_RULE_FILE}
+VPATH ?= $(shell pwd):$(PATH)
+# Runtime CLI to use for running images
+CONTAINER_ENGINE ?= $(shell command -v podman 2>/dev/null || echo docker)
+IMG_VERSION=1.0.4
 
-.PHONY: lint
-lint:
-	echo "Running Prometheus rules linter tests"
-	./promtool check rules ${EXTRACTED_RULE_FILE}
+BASE_CMD=$(CONTAINER_ENGINE) run -v "$(shell pwd):/work" --privileged -t quay.io/rhobs/obsctl-reloader-rules-checker:$(IMG_VERSION) -t rhtap -d rhobs/alerting -y -p
```

Comment:
Can we add a `--rm` here to delete the pod?

Reply:
Yes, thanks for the suggestion :)